htdocs/comm/index.php

```diff
@@ -168,10 +169,7 @@
 		$total = 0;
 		$num = $db->num_rows($resql);
 
-		print '<div class="div-table-responsive-no-min">';
-		print '<table class="noborder centpercent">';
-		print '<tr class="liste_titre">';
-		print '<th colspan="3">'.$langs->trans("ProposalsDraft").' <a href="'.DOL_URL_ROOT.'/comm/propal/list.php?search_status=0"><span class="badge">'.$num.'</span></a></th></tr>';
+		StartSimpleTableHeader("ProposalsDraft", "comm/propal/list.php", "search_status=0", 2, $num);
```

Comment:
Introducing new function need to follow the camelCase format. Can you rename into 
startSimpleTableHeader
(first s is lowercase)